packages/components/src/CollapsiblePanel/CollapsiblePanel.scss

```diff
@@ -62,31 +66,125 @@ $tc-collapsible-panel-btn-color: #555964;
 	}
 }
 
-.group {
-	justify-content: space-between;
-
-	> * {
-		margin: 0 $tc-collapsible-panel-padding-normal;
+.tc-panel {
+	:global(.panel-heading) {
+		@include header-style;
+		padding-right: $tc-collapsible-panel-padding-larger + $tc-collapsible-panel-padding-normal;
 	}
+}
 
-	:global(.label) {
-		white-space: nowrap;
-		text-overflow: ellipsis;
-		overflow: hidden;
-		font-size: 1em;
+.tc-selectable-panel {
+	border-radius: 0;
+	margin-bottom: 0;
+
+	&.selected-panel {
+		border-left: 4px solid #6aa9d1;
 	}
-}
 
-.tc-panel {
 	:global(.panel-heading) {
-		@include header-style;
-		padding-right: $tc-collapsible-panel-padding-larger + $tc-collapsible-panel-padding-normal;
+		display: flex;
+		justify-content: space-between;
+		background-color: white;
+		border-color: transparent;
+		padding: 0;
+
+		.header-title-class {
+			color: $tc-collapsible-selectable-panel-title-color;
+			font-weight: 600;
+			overflow: hidden;
+			text-overflow: ellipsis;
+			width: 70px;
+			white-space: nowrap;
+		}
+
+		.header-tag-class {
+			color: $tc-collapsible-selectable-panel-content-description-color;
+			margin-left: 0;
+			white-space: nowrap;
+		}
+
+		.header-detail-class {
```

Comment:
header-title-class, header-tag-class, header-detail-class
Please rename that to be simpler (title, tag, detail)
Those should be available for all headers